recipes/configure.rb

```diff
@@ -37,6 +37,7 @@
   fire_hook(:configure, items: databases + [source, framework, appserver, worker, webserver])
 
   execute 'monit reload' do
+    action :nothing
```

Comment:
Don't reload monit during a config run. If one of the config files change, they will notify this that a reload is needed.